code-challenges.test.js

```diff
@@ -15,43 +15,97 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// describe ("removeShuff", () => {
+//   it("returns the array with the first item gone and the rest of the array shuffled.", () => {
+//     expect.not.arrayContaining(colors1)
+//     expect.not.arrayContaining(colors1)
+//   })
+// })
 
-// b) Create the function that makes the test pass.
+// const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+
+
+// const colors2 = [
+//   "chartreuse",
+//   "indigo",
+//   "periwinkle",
+//   "ochre",
+//   "aquamarine",
+//   "saffron"
+// ]
+
+// const removeShuff = (array) => {
+//   return (array.splice(1, 5).sort((a, b) => 0.5 - Math.random(array)))
+// }
 
-// Pseudo code:
+
+// console.log(removeShuff(colors1))  --> Left these in because the test passes even with no function
+// console.log(removeShuff(colors2))
+// ^^^^^^^^^  Left these in because the test passes even with no function
+
+// Pseudo code: Input: array  Output: first item moved from array and shuffled
+// Create a function that takes in the array as a parameter/argument. Use.splice() to remove the first item from the array (starting the index at 1 and going up to 5 to include the final index of colors 2 as well. .sort the new array with an action to a 0.5 float, minused off the result of Math.random.
```

Comment:
Your test passes without a function because you are not performing a function call in the expect statement, which means the test is not testing for anything.
This explanation of splice is not accurate, I think you're confusing it with slice. Also, Math.random() does not need to have an argument in the parentheses.
Great thought process and approach, just work on understanding what's going on and explaining it properly.